app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,25 +3,58 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_VALUES = {
   title: "",
   description: "",
-  assignedContactId: "",
-  tag: "",
+  assignedContact: [],
+  tag: [],
```

Comment:
How does assigning a singular entity, that is `tag`, with a collection like array make sense? If it's something that can hold a collection then name it appropirately.